test/Aeson.purs

```diff
@@ -15,18 +18,30 @@ import Aeson
   , parseJsonStringToAeson
   , toObject
   , toStringifiedNumbersJson
+  , stringifyAeson
+  , getNumberIndex
   )
-import Data.Argonaut (encodeJson, parseJson, stringify)
-import Data.Array (head)
+import Control.Apply (lift2)
+import Control.Monad.Cont (lift)
+import Data.Argonaut (encodeJson, parseJson)
+import Data.Argonaut as Json
+import Data.Array (head, length, (!!), zip)
 import Data.BigInt as BigInt
-import Data.Either (Either(Right))
+import Data.Either (Either(..))
```

Comment:
I think we've also settled on using explicit constructor imports (there are a few more implicit imports below)